frontend/src/utils/common.js

```diff
@@ -52,6 +52,25 @@ export async function downloadResource(url) {
 	}
 }
 
+export async function downloadPublicResource(url) {
```

Comment:
could we just reuse the downloadResource function? the content is exactly the same except leave out the "authHeader"

Reply:
I tested and I could use downloadResource, removed the other